benchmarks/off-policy/README.md

```diff
@@ -1,40 +1,56 @@
-# OmniSafe's Mujoco Velocity Benchmark on Off-Policy Algorithms
+# OmniSafe's Mujoco Velocity Benchmark for Off-Policy Algorithms
 
-OmniSafe's Mujoco Velocity Benchmark evaluated the performance of OmniSafe algorithm implementations in 6 environments from the Safety-Gymnasium task suite For each algorithm and environment supported, we provide:
-- Default hyperparameters used for the benchmark and scripts to reproduce the results
-- A comparison of performance or code-level details with other open-source implementations or classic papers.
-- Graphs and raw data that can be used for research purposes, - Log details obtained during training
+The OmniSafe Mujoco Velocity Benchmark for off-policy algorithms evaluates the effectiveness of OmniSafe's off-policy algorithms across multiple environments from the [Safety-Gymnasium](https://github.com/PKU-Alignment/safety-gymnasium) Mujoco Velocity task suite. For each algorithm and environment supported, we provide:
+
+- Default hyperparameters used for the benchmark and scripts to reproduce the results.
+- A comparison of performance or code-level details with other open-source implementations.
```

Comment:
why do you del this line in model-based benchmark but add in here?

Reply:
Because model-based algorithms did not make comparison with other open-source implementations, but on-policy and off-policy did.